### PR TITLE
Fixes edge case where it was possible to get multiple parasites without admin intervention

### LINKED ISSUE #16490: You can use a guardian creation thing multiple times
1. Player got two in a surplus kit. Used both.

2. One player got two from the same one. I assume they just spam clicked or something.

I assume hasparasites isn't tested (though it'd be a massive fucking pain to test by yourself to be fair)

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #16490

### CODE REVIEW

**Comment by Robustin** [comment]
And I'm about to open a new PR to let people do this shit by default. Just FYI.

Lings and Traitor memes is one debate, but multiple HP's presents no unique balance issues. It's a rare case and the cost (getting screwed out of 14TC, prohibiting what could be a very interesting fight) doesn't justify whatever nebulous benefit exists for this rule.

**Comment by lzimann** [comment]
It was already opened once, and it was declined.

**Comment by ChangelingRain** [comment]
gET YOUR ARGUMENT OUT OF MY BUGFIX PR

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -415,6 +415,11 @@ var/global/list/parasites = list() //all currently existing/living guardians
 		if("Assassin")
 			pickedtype = /mob/living/simple_animal/hostile/guardian/assassin
 
+	var/list/guardians = user.hasparasites()
+	if(guardians.len && !allowmultiple)
```
Comment (Jordie0608) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Why not just `if(user.hasparasites() && !allowmultiple)`

Comment (Jordie0608) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Yay byond

**Comment by Jordie0608** [comment]
Apparently it somehow did but I don't think it's worth trying to patch around admins messing with vars so all's good

File: code/modules/mob/living/simple_animal/guardian/guardian.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: of course it's true.
```if(object)``` is true, and a list is a datastructure, it's an object.

**Pull request merged**